SCHIZO/Creatures/Ermfish/ErmfishLoader.cs

```diff
@@ -77,7 +77,8 @@ private static void LoadErmfish()
         });
 
         List<LootDistributionData.BiomeData> biomes = new();
-		foreach (BiomeType biome in Enum.GetValues(typeof(BiomeType)).Cast<BiomeType>())
+		foreach (BiomeType biome in Enum.GetValues(typeof(BiomeType)).Cast<BiomeType>()
+            .Where(biome => biome.ToString().EndsWith("CreatureOnly")))
```

Comment:
What's going on here?

Reply:
There are 495 enum values in all and most of them don't actually represent a good spawn location for any kind of creature - wrecks, walls, floors, etc. Most common fish have about 20-40 biomes assigned to them, and this change brings the previous 495 down to ~50. I've been playing this branch on my save over the weekend and haven't noticed too big of a change in spawn rates - ermfish and ermsharks both still show up everywhere

Reply:
I just messed around with spawn biomes tonight and the ones Gov is picking are perfect for fish. They're basically always two different levels of open water (deep or shallow). You're guaranteed not to have accidental in-terrain spawns, and I believe those zones are specifically designated for fish to spawn. Might have to increase/tweak probabilities, but this is likely better than trying to spawn in all biomes.